### PR TITLE
Milestone4/logic

### PR DESCRIPTION
**Pull request by epinczinger**
On this branch, I've worked on the following specifics:

- Create App Logics.
- Add Operate component.
- Add Calculate component.
- Style of the app, implemented with Tailwind, following a responsive design.
- Eslint errors have been fixed.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -5,8 +5,10 @@ import Display from './Display';
 export default function App() {
```
Comment (wathigo) [inline src/components/App.js:6]: **REQUIRED** 
* calculate.js is imported in the App component.
Screenshot from 2021-03-25 16-36-18

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this milestone.
Let's fix one issue before we move on. See the comment :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Comment by epinczinger** [comment]
Calculate is being imported on App but is commented on in order to avoid Eslint errors.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @epinczinger, congratulations on this project :fireworks: Having imported the calculate.js file in the app component as outlined in the previous review :muscle: I hereby approve the project :+1: 

Kindly go ahead and merge into development :+1: Do not forget to submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**